docs/api/browser-window.md

```diff
@@ -264,6 +264,9 @@ It creates a new `BrowserWindow` with native properties as set by the `options`.
       canvas features. Default is `false`.
     * `scrollBounce` Boolean (optional) - Enables scroll bounce (rubber banding) effect on
       macOS. Default is `false`.
+    * `tabbingIdentifier` String (optional) - Tab group name, allows opening the
```

Comment:
This looks like it is under the `webPreferences` outer list but it isn't a web preference.

Perhaps it should go after `zoomToPageWidth` instead.